fit.py

```diff
@@ -0,0 +1,310 @@
+import os
+import pprint
+
+import pymongo
+from bson.objectid import ObjectId
+from dotenv import load_dotenv
+from flask import (
+    Flask,
+    flash,
+    redirect,
+    render_template,
+    request,
+    session,
+    url_for,
+)
+from flask_login import LoginManager
+from utilities import paginate
+from validators import validate_recipe
+
+load_dotenv()
+
+app = Flask(__name__)
+app.config["SECRET_KEY"] = "qwert"
+login = LoginManager(app)
+
+
+myclient = pymongo.MongoClient(os.environ.get("MONGO_CLUSTER"))
+mydb = myclient["fitness_pot"]
+mycol = mydb["dish"]
```

Comment:
Use the find and replace feature on your terminal  to replace mycol to `dish_cos`